--- sysprof-3.36.0/src/tests/allocs-by-size.c.orig	2020-03-06 23:04:34.336270600 +0100
+++ sysprof-3.36.0/src/tests/allocs-by-size.c	2020-04-15 15:21:01.438421529 +0200
@@ -109,7 +109,7 @@
     {
       const Item *item = &g_array_index (ar, Item, i);
 
-      g_print ("%"G_GUINT64_FORMAT",%"G_GUINT64_FORMAT",%"G_GUINT64_FORMAT"\n",
+      g_print ("%"G_GSIZE_FORMAT",%"G_GSIZE_FORMAT",%"G_GSIZE_FORMAT"\n",
                item->size, item->cmp, item->count);
     }
 }
